bokehjs/src/lib/models/tools/gestures/box_zoom_tool.ts

```diff
@@ -253,7 +254,7 @@ export class BoxZoomTool extends GestureTool {
     this.prototype.default_view = BoxZoomToolView
 
     this.define<BoxZoomTool.Props>(({Bool, Ref, Or, Auto}) => ({
-      dimensions:   [ Or(Dimensions, Auto), "both" ],
+      dimensions:   [ Or(Dimensions, Auto), "auto" ],
```

Comment:
This change has to be applied in bokeh (Python) as well and then deafults' baseline file regenerated with `pytest tests/test_defaults.py`. I'm surprised this wasn't caught by bokehjs' defaults tests. I will need to investigate this.

Reply:
This is a bug in the runner of defaults' tests.

Reply:
Fixed by PR #14102.